README.md

```diff
@@ -1,18 +1,18 @@
 # PowerVS Infrastructure Module
 
 The PowerVS infrastructure module automates the following tasks:
-- Creates the PowerVS service
+- Creates the PowerVS Workspace
 - Creates an ssh key
 - Creates two private networks: a management network and a backup network
 - Creates two IBM Cloud connections with an option to reuse cloud connections
 - Attaches the IBM Cloud connections to a transit gateway
 - Attaches the private networks to the IBM Cloud connections
 
 Following limitations currently apply:
-- The only number of IBM Cloud connections we support is two.
+- The only number of IBM Cloud connections we support is **two**.
 - Reuse of IBM Cloud connections is not supported.
-- Private networks in IBM PowerVS service must be in 10.0.0.0/8 range
-- Only SLES15 SP3 as operating system is supported
+- Private networks in IBM PowerVS Workspace must be in 10.0.0.0/8 range
+- **Only SLES15 SP3 and RHEL 8.4 as operating system is supported**
```

Comment:
Only SLES15 SP3 and RHEL 8.4 as operating system **are** supported

Reply:
Updated